drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -93,6 +93,1796 @@ matterManufacturer:
     vendorId: 0x100b
```

Comment:
Note: this `0x2168` fingerprint existed already but it also appeared in the device spreadsheet, so I removed the new fingerprint to keep this original one.